edisgo/tools/config.py

```diff
@@ -129,6 +138,102 @@ def __init__(self, **kwargs):
                 filename=kwargs.get("json_filename", None),
                 from_zip_archive=kwargs.get("from_zip_archive", False),
             )
+        self._config_dict = {}
+
+    @property
+    def db_table_mapping(self):
+        if not self._config_dict.get("db_table_mapping"):
+            self._set_db_mappings()
+        return self._config_dict.get("db_table_mapping", {})
+
+    @db_table_mapping.setter
+    def db_table_mapping(self, value):
+        self._config_dict["db_table_mapping"] = value
+
+    @property
+    def db_schema_mapping(self):
+        if not self._config_dict.get("db_schema_mapping"):
+            self._set_db_mappings()
+        return self._config_dict.get("db_schema_mapping", {})
+
+    @db_schema_mapping.setter
+    def db_schema_mapping(self, value):
+        self._config_dict["db_schema_mapping"] = value
+
+    def _set_db_mappings(self) -> None:
+        """
+        Sets the database table and schema mappings by retrieving alias dictionaries.
+        """
+        name_mapping, schema_mapping = self.get_database_alias_dictionaries()
+        self.db_table_mapping = name_mapping
+        self.db_schema_mapping = schema_mapping
+
+    def get_database_alias_dictionaries(self) -> tuple[dict[str, str], dict[str, str]]:
+        """
+        Retrieves the database alias dictionaries for table and schema mappings.
+
+        Returns
+        -------
+        tuple
+            A tuple containing two dictionaries:
+            - name_mapping: A dictionary mapping source table names to target table
+                names.
+            - schema_mapping: A dictionary mapping source schema names to target schema
+                names.
+        """
+        OEP_CONNECTION = "postgresql+oedialect://:@{platform}"
+        platform = "toep.iks.cs.ovgu.de"
+        conn_str = OEP_CONNECTION.format(platform=platform)
+        engine = sa.create_engine(conn_str)
+        schema_name = "model_draft"  # Replace with the actual schema name if needed
```

Comment:
Shouldn't `schema_name` be a method variable?

Reply:
this is just for the dictionary table, that "path" is fixed and doesn't change (hopefully)

Reply:
I renamed the variables for improved readability